### PR TITLE
Cirrus: CI branch updates

### PR DESCRIPTION
**Pull request by cevich**
This Depends on https://github.com/containers/netavark/pull/423 merging and [a branch-build running and passing](https://cirrus-ci.com/github/containers/netavark/v1.1.0-rhel).

### CODE REVIEW

**Comment by lsm5** [comment]
> This Depends on #234 merging and [a branch-build running and passing](https://cirrus-ci.com/github/containers/netavark/v1.1.0-rhel).

this is self-referencing unless you meant a different PR

File: Makefile [hunk 1]
```diff
@@ -98,7 +98,6 @@ mock-rpm:
 .PHONY: validate
 validate: $(CARGO_TARGET_DIR)
 	cargo fmt --all -- --check
-	cargo clippy -p aardvark-dns -- -D warnings
```
Comment (lsm5) [inline Makefile:101]: does cargo fmt handle clippy too ?

**Comment by Luap99** [comment]
LGTM but we should really pin the rust version to prevent the lint problems, especially on the maintenance branches.

**Comment by cevich** [comment]
> this is self-referencing unless you meant a different PR

Woops...okay I fixed it.

> LGTM but we should really pin the rust version to prevent the lint problems, especially on the maintenance branches.

Yes, agreed, there is [an open issue for this](https://github.com/containers/automation_images/issues/150) so it shouldn't be forgotten.

**Review by Luap99 — APPROVED** [review]
/lgtm

**Pull request merged**